tasks/pipeline.py

```diff
@@ -582,21 +579,16 @@ def changelog(ctx, new_commit_sha):
     )
 
 
-def _init_pipeline_schedule_task():
-    gitlab = Gitlab(api_token=get_gitlab_bot_token())
-    gitlab.test_project_found()
-    return gitlab
-
-
 @task
 def get_schedules(_):
     """
     Pretty-print all pipeline schedules on the repository.
     """
 
-    gitlab = _init_pipeline_schedule_task()
-    for ps in gitlab.all_pipeline_schedules():
-        pprint.pprint(ps)
+    repo = get_gitlab_repo(token=get_gitlab_bot_token())
+
+    for sched in repo.pipelineschedules.list(per_page=100, all=True):
+        sched.pprint()
```

Comment:
Is there really a `pprint` method in the returned object of `pipelineschedules` (so [this one](https://python-gitlab.readthedocs.io/en/stable/api/gitlab.v4.html#gitlab.v4.objects.ProjectPipelineSchedule) I guess)?
Or I misunderstood something?
Same remark on the chunks below.
```suggestion
        pprint.pprint(sched)
```
Even if I'm not sure what `pprint.pprint` will do in that kind of object